src/OrleansTelemetryConsumers.Counters/OrleansPerfCounterTelemetryConsumer.cs

```diff
@@ -10,61 +10,26 @@ namespace OrleansTelemetryConsumers.Counters
     /// <summary>
     /// Telemetry consumer that writes metrics to predefined performance counters.
     /// </summary>
-    public class OrleansPerfCounterTelemetryConsumer : IMetricTelemetryConsumer
+    internal class OrleansPerfCounterTelemetryConsumer : IMetricTelemetryConsumer
```

Comment:
Since it needs SiloAssemblyLoader in the ctor, but that's internal. I'm not certain what the best solution is here. Need to work something out...